project/AppDependencies.scala

```diff
@@ -1,38 +1,32 @@
 
 import play.core.PlayVersion.current
-import sbt._
+import sbt.*
 
 object AppDependencies {
 
-  val compile: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "7.1.0",
-    "uk.gov.hmrc" %% "bootstrap-backend-play-28" % "7.1.0",
-    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28" % "0.71.0",
-    "uk.gov.hmrc" %% "play-frontend-hmrc" % "3.23.0-play-28",
-    "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.13.3"
-  )
+  private val bootstrapPlayVersion = "7.1.0"
 
-  def commonTestDependencies(scope: Configuration): Seq[ModuleID] = Seq(
-    "org.scalatestplus.play" %% "scalatestplus-play" % "5.1.0" % scope,
-    "org.scalatest" %% "scalatest" % "3.1.1" % scope,
-    "org.jsoup" % "jsoup" % "1.14.1" % scope,
-    "com.typesafe.play" %% "play-test" % current % scope,
-    "com.vladsch.flexmark" % "flexmark-all" % "0.36.8" % scope,
-    "uk.gov.hmrc.mongo" %% "hmrc-mongo-test-play-28" % "0.71.0" % scope
+  val compile: Seq[ModuleID] = Seq(
+    "uk.gov.hmrc"                  %% "bootstrap-frontend-play-28" % bootstrapPlayVersion,
+    "uk.gov.hmrc"                  %% "bootstrap-backend-play-28"  % bootstrapPlayVersion,
+    "uk.gov.hmrc.mongo"            %% "hmrc-mongo-play-28"         % "0.71.0",
```

Comment:
This can be bumped to `0.74.0` and kept in sync with its test library using a `private val`, like Bootstrap.